com.biglybt.BiglyBT.yaml

```diff
@@ -26,10 +26,12 @@ modules:
     build-commands:
       - mv biglybt /app/biglybt
       - install -dm755 /app/bin
+      - install -m755 biglybt-runner /app/bin/biglybt-runner
+      - install -dm755 ~/.biglybt/custom
+      - install -m755 custom-presets.config ~/.biglybt/custom/custom-presets.config
```

Comment:
this definitely won'y work. the $HOME is NEVER included in a flatpak.

Reply:
Thanks for your feedback, yes I changed this and was testing locally but got interrupted earlier, just pushed the changes